docs/core/tools/dotnet-build.md

```diff
@@ -18,13 +18,13 @@ ms.date: 05/25/2018
 # [.NET Core 2.x](#tab/netcore2x)
 ```
 dotnet build [<PROJECT>] [-c|--configuration] [-f|--framework] [--force] [--no-dependencies] [--no-incremental]
-    [--no-restore] [-o|--output] [-r|--runtime] [-v|--verbosity] [--version-suffix]
+    [--no-restore] [-o|--output] [-r|--runtime] [-v|--verbosity] [--version-suffix] [-p|-property] [-l|-logger]
```

Comment:
@ryzngard if we go this way, *all* MSBuild options should be added to the synopsis. That would make this section huge. Also, I think the original intention was to list the options of *dotnet build* command itself.

Reply:
I wanted to be consistent with the wording we have later 

> such as `-p` for setting properties or `-l` to define a logger.

Maybe not necessary, but that was the intent. I'm happy to remove this and the notes in the arguments section below, leaving the header for MSBuild to signify the capabilities.

Reply:
>I'm happy to remove this and the notes in the arguments section below, leaving the header for MSBuild to signify the capabilities.

That looks good to me.